tests/integ/test_packaging.py

```diff
@@ -966,3 +966,33 @@ def test_get_available_versions_for_packages(session):
     assert returned.keys() == set(packages)
     for key in returned.keys():
         assert len(returned[key]) > 0
+
+
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC or IS_WINDOWS,
+    reason="Subprocess calls are not allowed within stored procedures. Custom package upload does not work well on Windows.",
+)
+def test_replicate_local_environment(session):
```

Comment:
I don't understand what this test is testing. the sample udf does not need any packages to work so what's the point of replicating local environment here?

Reply:
This is true, It is difficult to know how to test replicate_local_environment in a non-flaky way because it is tied to the environment (I would have to pip install stuff in the environment to make a valid test).

This test only proves that replicate_local_environment does not prevent UDF from working, not super useful.

Reply:
Hmm I see. This is interesting. Is there a package that we absolutely will install in the enviornment that we could do an import test? 🤔

Reply:
`cloudpickle` maybe?